run_targets_1718.R

```diff
@@ -0,0 +1,20 @@
+library(targets)
+
+Sys.setenv("CREATESLF_KEYRING_PASS" = "createslf")
+
+year <- "1718"
+
+# use targets for the process until testing episode files
+tar_make_future(
+  # it does not recognise `contains(year)`
+  names = (targets::contains("1718"))
+)
+
+# use targets to create individual files due to RAM limit
+library(createslf)
+
+episode_file <- arrow::read_parquet(get_slf_episode_path(year))
+
+# Run individual file
+create_individual_file(episode_file, year = year) %>%
+  process_tests_individual_file(year = year)
```

Comment:
I think overall this PR is ready to merge. However i have some questions about the set up here. When you are back from A/L can we have a call to discuss? 

I wonder if we need to have separate set up scripts for the individual file if this runs out of memory during the processing? 

There are also workbench job scripts in the folder Run_SLF_Files_targets. so i think these would need to go into that folder to replace each `year` script.